addons/sale/views/res_partner_views.xml

```diff
@@ -16,6 +16,22 @@
         </field>
     </record>
 
+    <record id="act_res_partner_2_sale_order_stat" model="ir.actions.act_window">
```

Comment:
I don't think there is a need to define a new action, we could update the existing one as it's the only place where it's used.

Reply:
Hello 

Thanks, for pointing it out i believe that's the same case for Customer Lots as well. There also a new action is not needed